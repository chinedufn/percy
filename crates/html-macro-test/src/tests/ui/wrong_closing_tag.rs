extern crate virtual_dom_rs;
use virtual_dom_rs::prelude::*;

// Expected a closing div tag, found a closing strong tag
fn main() {
    html! {
        <div> </strong>
    };
}
