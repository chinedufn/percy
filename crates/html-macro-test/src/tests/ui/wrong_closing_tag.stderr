error: Wrong closing tag. Try changing "strong" into "div"
 --> $DIR/wrong_closing_tag.rs:7:17
  |
7 |         <div> </strong>
  |                 ^^^^^^
